string             model_name
geometry_msgs/Pose module_pose
geometry_msgs/Pose left_pose
geometry_msgs/Pose right_pose
uint32             dist_cameras
float64            focal_length
uint32             height
uint32             width
